app/uk/gov/hmrc/gform/gform/EmailAuthController.scala

```diff
@@ -313,22 +316,33 @@ class EmailAuthController(
   ): Future[EmailAndCode] =
     formTemplate.authConfig match {
       case emailAuthConfig: EmailAuthConfig =>
-        val emailAndCode = EmailAndCode.emailVerificationCode(emailId.value.toString)
-        val emailVerifierService = emailAuthConfig.service match {
-          case Notify(notifierTemplateId) =>
-            EmailVerifierService.notify(NotifierTemplateId(notifierTemplateId.value))
-          case DigitalContact(emailTemplateId) =>
-            EmailVerifierService.digitalContact(EmailTemplateId(emailTemplateId.value))
+        val isDefaultEmailId = frontendAppConfig.emailAuthDefaultEmailIds.fold(false) {
+          _.map(id => ci"$id").contains(ci"${emailId.value.toString}")
         }
-        gformConnector
-          .sendEmail(
-            ConfirmationCodeWithEmailService(
-              NotifierEmailAddress(emailId.value.toString),
-              emailAndCode.code,
-              emailVerifierService
-            )
+
+        if (isDefaultEmailId) {
+          Future.successful(
+            EmailAndCode(ci"${emailId.value.toString}", EmailConfirmationCode(emailConfirmationCode))
           )
-          .map(_ => emailAndCode)
+        } else {
```

Comment:
I'd would prefer somehow more subtle approach. We don't need to have any code hardcoded etc. We can provide special RND to `emailVerificationCode ` which will give us deterministic code.

I did [POC of this approach](https://github.com/hmrc/gform-frontend/commit/dc36d8bfea28ec28c7cc99de0f0b8d1cd6a253a0).